services/store/store.go

```diff
@@ -1010,11 +1010,12 @@ func (s *Store) GetActivePipelineUsage(ctx context.Context, pipelineID string) (
 
 func (s *Store) WatchKeys(key string) chan *redis.Message {
 	ps := s.options.RedisBackend.PSubscribe(s.options.ShutdownCtx, "__keyspace@0__:"+key)
-	defer ps.Close()
 
 	keyChan := make(chan *redis.Message, 1)
 
 	go func() {
+		defer ps.Unsubscribe(s.options.ShutdownCtx, "__keyspace@0__:*"+key)
```

Comment:
Lol wtf is that key - can you link to docs about it + const?

Reply:
Done